libdnf/transaction/transaction_item_action.cpp

```diff
@@ -80,6 +82,9 @@ std::string TransactionItemAction_get_short(TransactionItemAction action) {
         case TransactionItemAction::REASON_CHANGE:
             // TODO(dmach): replace "?" with something better
             return "?";
+        case TransactionItemAction::REPLACED:
+            // TODO(jmracek): replace "?" with something better
+            return "?";
```

Comment:
While this will likely need revisiting, what about at least using a unique letter, e.g. "X", just in case it ever matters before we finalize these?

Reply:
What about to use "O" instead "X" or "?"?

Reply:
Sure. I assume we'll revisit these and figure out the best letters.

Reply:
Fixed